galaxy_ng/app/settings.py

```diff
@@ -21,6 +21,7 @@
 
 INSTALLED_APPS = [
     'rest_framework.authtoken',
+    'social_django',
```

Comment:
Maybe this must be added on dev/standalone-community env only?
adding here will require this app to be available for all intallations

Reply:
I agree, I'm just now sure how to do that. How did you inject that with the ldap mode? I notice it also uses the same library, which is why I picked it.

Reply:
Oh, I found it ... https://github.com/ansible/galaxy_ng/blob/facddc251698d1ac9cbd0dbb926956b6dbb428ca/galaxy_ng/app/dynaconf_hooks.py#L122-L123

I'll get that patched in ASAP.